lib/src/header_parser/sub_parsers/compounddecl_parser.dart

```diff
@@ -0,0 +1,303 @@
+// Copyright (c) 2020, the Dart project authors. Please see the AUTHORS file
```

Comment:
2021 (I guess that might be struct_parser but git not picking up the rename).

Reply:
Yes, it is. Since structs and unions were so similar, it seemed best to not have separate parsers for them.